README.md

```diff
@@ -342,19 +343,6 @@ export PINECONE_ENV="<YOUR_PINECONE_REGION>" # e.g: "us-east4-gcp"
 export MEMORY_BACKEND="pinecone"
 ```
 
-### Milvus Setup
```

Comment:
Don't remove milvus

Reply:
Milvus Setup is duplicated. You can see line 367 to 377